### ISSUE TITLE
Python 2 fails to install in gcc49

### ISSUE THREAD

**Issue by elizagamedev** [conan-io/conan-docker-tools#73]
This looks like a side effect of enforcing Python 3. If a `system_requirements()` call pulls in Python 2 in the dependency chain, the installation will fail with the output...

```Linking and byte-compiling packages for runtime python2.7...
Traceback (most recent call last):
  File "/usr/bin/pycompile", line 35, in <module>
    from debpython.version import SUPPORTED, debsorted, vrepr, \
  File "/usr/share/python/debpython/version.py", line 24, in <module>
    from ConfigParser import SafeConfigParser
ModuleNotFoundError: No module named 'ConfigParser'
dpkg: error processing package python2.7-minimal (--configure):
```

The problem is clear, but I'm not certain about the solution. Maybe pre-installing pycompile and patch it like lsb_release? The idea of doing that makes me wince a bit. Maybe it's sufficient to just install python2.7-minimal beforehand?

**Comment by elizagamedev** [comment]
I'm going to try building an image with python2.7-minimal installed before Python 3 and try building an offending package, sdl2/2.0.8@bincrafters/stable.

**Comment by Croydon** [comment]
This is exactly what https://github.com/bincrafters/conan-sdl2/pull/14 is blocking //cc @uilianries

**Comment by Croydon** [comment]
This is exactly what https://github.com/bincrafters/conan-sdl2/pull/14 is blocking //cc @uilianries

**Comment by uilianries** [comment]
So I messed up lsb_release when updating gcc48 and gcc49 images ... :sweat: 

I'll check soon, thanks for reporting.

**Comment by uilianries** [comment]
Hi @elizagamedev @Croydon !

I did some changes to fix both GCC 4.8 and GCC 4.9. It's working for SDL2 as well, but I would like your feedback before closing this issue.

**Comment by Croydon** [comment]
LGTM, thanks once again for your work @uilianries!
